packages/shorebird_cli/lib/src/config/shorebird_config_dir.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:cli_util/cli_util.dart';
+import 'package:meta/meta.dart';
+import 'package:shorebird_cli/src/command_runner.dart';
+
+final String shorebirdConfigDir = () {
```

Comment:
Why the closure?  Just to get it to run at init time rather than compile time I guess?